system-variables.md

```diff
@@ -142,12 +166,44 @@ mysql> SELECT * FROM t1;
 - This variable sets the authentication method that the server advertises when the server-client connection is being established. Possible values for this variable are documented in [Authentication plugin status](/security-compatibility-with-mysql.md#authentication-plugin-status).
 - Value options: `mysql_native_password` and `caching_sha2_password`. For more details, see [Authentication plugin status](/security-compatibility-with-mysql.md#authentication-plugin-status).
 
+### default_week-format
+
+- Scope: SESSION | GLOBAL
+- Default value: `0`
+- Range: `[0, 7]`
+- Set the week format used by the `WEEK()` function
+
+### error_count
+
+- Scope: SESSION
+- Default value: `0`
+- This variable exists only for MySQL compatibility and is always `0` in TiDB.
```

Comment:
It looks like there is code in the server for this feature, but it might not be working correctly?

Reply:
```
mysql-8.0.26 > select now(999999);
ERROR 1426 (42000): Too-big precision 63 specified for 'now'. Maximum is 6.
mysql-8.0.26 > SELECT @@session.error_count;
+-----------------------+
| @@session.error_count |
+-----------------------+
|                     1 |
+-----------------------+
1 row in set (0.00 sec)
```

```
5.7.25-TiDB-v5.0.4 127.0.0.1:4000  test  SQL  select now(999999);
ERROR: 1105 (HY000): Too-big precision 999999 specified for 'now'. Maximum is 6.
5.7.25-TiDB-v5.0.4 127.0.0.1:4000  test  SQL  SELECT @@session.error_count;
+-----------------------+
| @@session.error_count |
+-----------------------+
| 0                     |
+-----------------------+
1 row in set (0.0014 sec)
```

```
8.0.26 127.0.0.1:8026   test  SQL  select now(999999);
ERROR: 1426 (42000): Too-big precision 63 specified for 'now'. Maximum is 6.
8.0.26 127.0.0.1:8026   test  SQL  SELECT @@session.error_count;
+-----------------------+
| @@session.error_count |
+-----------------------+
|                     0 |
+-----------------------+
1 row in set (0.0007 sec)
```

```
mysql> select now(999999);
ERROR 1105 (HY000): Too-big precision 999999 specified for 'now'. Maximum is 6.
mysql> SELECT @@session.error_count;
+-----------------------+
| @@session.error_count |
+-----------------------+
| 0                     |
+-----------------------+
1 row in set (0.00 sec)
```

Looks like this doesn't work with MySQL Shell, but also not with TiDB.